samples/contacturi/src/main/java/com/bumptech/glide/samples/contacturi/MainActivity.java

```diff
@@ -40,6 +48,17 @@ protected void onCreate(Bundle savedInstanceState) {
     imageViewLookup = (ImageView) findViewById(R.id.image_lookup);
     imageViewPhoto = (ImageView) findViewById(R.id.image_photo);
     imageViewDisplayPhoto = (ImageView) findViewById(R.id.image_display_photo);
+    numberEntry = (EditText) findViewById(R.id.number_entry);
+    if (ContextCompat.checkSelfPermission(this.getApplication(),
```

Comment:
The spacing looks a bit weird, maybe:

if (ContextCompat.checkSelfPermission(
    getApplication(), 
   Manifeset.permission.READ_CONTACTS) != PackageManager.PERMISSION_GRANTED) {
...
}